### PR TITLE
test:ICalExporter test added

### PR DESCRIPTION
**Pull request by mrsaicharan1**
helps in #5320

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added unit test for ICalExporter helpers file

### CODE REVIEW

File: app/factories/common.py [hunk 1]
```diff
@@ -7,10 +8,10 @@
 integer_ = 25
 url_ = static_page
 imageUrl_ = image_link
-date_ = '2016-12-13T23:59:59.123456+00:00'
-dateFuture_ = '2099-12-13T23:59:59.123456+00:00'
-dateEndFuture_ = '2099-12-14T23:59:59.123456+00:00'
-dateEnd_ = '2020-12-14T23:59:59.123456+00:00'
+date_ = datetime.datetime(2016, 12, 13)
```
Comment (iamareebjamal) [inline app/factories/common.py]: Why was this change needed?

Comment (mrsaicharan1) [reply app/factories/common.py]: Screenshot from 2019-03-21 03-09-33

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review
